third_party/tlx/dialect/include/IR/Dialect.h

```diff
@@ -23,6 +23,7 @@ constexpr static char AttrHasExplicitLocalMemAccessName[] =
     "tlx.has_explicit_local_mem_access";
 constexpr static char AttrHasTLXOpsName[] = "tlx.has_tlx_ops";
 constexpr static char AttrHasWarpSpecOpsName[] = "tlx.has_warp_spec_ops";
+constexpr static char AttrIsTLXTwoCTAModeName[] = "tlx.is_tlx_two_cta_mode";
```

Comment:
nit: how about rename this as `tlx.enable_paired_cta_mma` or something like that?